utils/test/i18n/i18n.util.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+import { ReactNode, ReactElement } from 'react';
+import { isElement } from 'react-dom/test-utils';
+
+const stringify: (node: ReactNode) => string = (node) => {
+  if (node === undefined || node === null) {
```

Comment:
Can this just be replaced with `if (!node) {...}`

Reply:
Fair point